src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -118,4 +131,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         $io->newLine();
     }
+
+    private function getFileLink(string $class)
+    {
+        try {
+            $r = new \ReflectionClass($class);
```

Comment:
we should use `ContainerBuilder::getReflectionClass($class, false)` instead, it handles more cases.

Reply:
Fixed :)